src/components/Channel/ChatForm.jsx

```diff
@@ -0,0 +1,83 @@
+import { useState, useEffect } from 'react';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+
+import { socketClient } from '../../hooks/socket/useSocket';
+
+export default function ChatForm({ channelId }) {
+  const [input, setInput] = useState('');
+  const [userName, setUserName] = useState('');
+
+  useEffect(() => {
+    const { name } = JSON.parse(sessionStorage.getItem('user'));
+    setUserName(name);
+  }, []);
+
+  function submitChat(ev) {
+    ev.preventDefault();
+
+    const trimmedInput = input.trim();
+
+    if (!trimmedInput) return;
```

Comment:
좀 더 명시적으로
```suggestion
    if (trimmedInput === '') return;
```
이렇게 어떤 경우에 return을 할건지 나타내주면 좋을 것 같아요.